Nature reserve boundaries revision

vholten opened the pull request:
Related to #3538, #2978

Changes proposed in this pull request:
- Make nature reserve boundaries less prominent at z8 and z9
- On all zoom levels: Reduce the "glowing" effect that occurs on boundaries that are shared between two adjacent nature reserves

Test renderings of [Northern California](https://www.openstreetmap.org/#map=8/41.263/-122.574)

z8 before:
![z8osm](https://user-images.githubusercontent.com/5209216/50043955-36188780-007d-11e9-9158-96a7ec4bf12b.png)

z8 after:
![z8tuned](https://user-images.githubusercontent.com/5209216/50424607-32e47f00-0867-11e9-940a-a705a193233f.png)

z9 before:
![z9osm](https://user-images.githubusercontent.com/5209216/50043958-403a8600-007d-11e9-9e64-3ec56f802a23.png)

z9 after:
![z9tuned](https://user-images.githubusercontent.com/5209216/50424610-3b3cba00-0867-11e9-9c9f-fdc9349c4671.png)

z10 before:
![z10osm](https://user-images.githubusercontent.com/5209216/50043960-492b5780-007d-11e9-975b-ae48360adde4.png)

z10 after:
![z10tuned](https://user-images.githubusercontent.com/5209216/50424613-40016e00-0867-11e9-94cd-52adf9c30aa9.png)

Tomasz-W commented:
@vholten Can you try with 1.3 width on z8 and 1.6 on z9? When I was checking test images on my mobile phone, these borders were quite hard to see.

vholten commented:
OK, here are the test renders with 1.3 width on z8 and 1.6 on z9. The differences are very small however. To tune the prominence of the borders at these zoom levels, it is better to change the opacity (currently 0.2 in this PR at z8 and z9).

z8, width 1.3:
![z8new13](https://user-images.githubusercontent.com/5209216/50044331-4bdc7b80-0082-11e9-8e7d-6b83ce4ca4a3.png)

z9, width 1.6:
![z9new16](https://user-images.githubusercontent.com/5209216/50044333-4f700280-0082-11e9-84d2-49c664a937a7.png)

Tomasz-W commented:
Thanks! Please test also this place where these areas are not only an outiline over bare ground
https://www.openstreetmap.org/#map=8/48.897/20.599

Adamant36 commented:
Thanks for doing this @vholten. It looks a lot better.

Tomasz-W commented:
I think they are a little bit too thick on z10 also (e.g. https://www.openstreetmap.org//#map=10/51.4326/13.3690), so I propose a scale with:
* 1.2 on z8
* 1.5 on z9
* 1.8 on z10

matkoniecz commented:
@Tomasz-W Can you give some test rendering for this proposed scale?

vholten commented:
@matkoniecz I plan to show test renderings for this PR and for the scale proposed by @Tomasz-W in the area of https://www.openstreetmap.org/#map=8/48.897/20.599 within the next couple of days.

vholten commented:
I have done some tests in the region suggested by @Tomasz-W and based on the results I think that the borders should be made a bit stronger than in my first proposal on z8 and z9  (opacity of 0.25 instead of 0.2).
On z10 I have tested thinner borders (width of 1.8 instead of 2).

(All test renders also use #3553 and #3563.)

z8
--

z8 before:
![z8borders](https://user-images.githubusercontent.com/5209216/50290753-f7b00d80-046c-11e9-855b-0d1750a42ff9.png)

z8 first proposal (opacity 0.2):
![z8pr](https://user-images.githubusercontent.com/5209216/50290762-00a0df00-046d-11e9-9ed4-298b16b77738.png)

z8 proposed stronger borders (opacity 0.25):
![z8pr-op25](https://user-images.githubusercontent.com/5209216/50290790-16160900-046d-11e9-92f0-021f21a4f169.png)

z9
--

z9 before:
![z9borders](https://user-images.githubusercontent.com/5209216/50290865-4067c680-046d-11e9-91f8-bd3195c1e327.png)

z9 first proposal (opacity 0.2):
![z9pr](https://user-images.githubusercontent.com/5209216/50290871-4362b700-046d-11e9-8c9d-f712464a3fde.png)

z9 proposed stronger borders (opacity 0.25):
![z9pr-op25](https://user-images.githubusercontent.com/5209216/50290874-465da780-046d-11e9-9f4c-c8107fc18c1b.png)

z10
--

z10 before:
![z10borders](https://user-images.githubusercontent.com/5209216/50290955-80c74480-046d-11e9-9a2a-261223c7e9d2.png)

z10 first proposal (thickness 2.0):
![z10pr](https://user-images.githubusercontent.com/5209216/50290976-8b81d980-046d-11e9-88a6-15cead4e3371.png)

z10 thinner borders (thickness 1.8):
![z10pr-w1 8](https://user-images.githubusercontent.com/5209216/50290995-99375f00-046d-11e9-9b9d-1a7d862dc83c.png)

kocio-pl commented:
When you update the code I guess it will be ready to merge.

vholten commented:
I've updated the code with the proposed changes. These are:
- Slightly more prominent borders on on z8 and z9 (opacity of 0.25 instead of 0.20)
- Thinner borders on z10 (width of 1.8 instead of 2.0)

I've also updated the test renderings of Northern California in the first post.

What do people think about making borders thinner on z11 as well?

vholten commented:
z11 current PR (https://www.openstreetmap.org/#map=11/49.5525/20.1495):
![z11before](https://user-images.githubusercontent.com/5209216/50446462-52d27c00-0915-11e9-8779-9e7d36b80d71.png)

z11 thinner borders (width of 1.8 instead of 2.0)
![z11-w1 8](https://user-images.githubusercontent.com/5209216/50446530-95945400-0915-11e9-8ff5-7b7702cff910.png)

kocio-pl commented:
Works for me.

vholten commented:
I've updated the code, I guess it's ready to be merged.

kocio-pl commented:
Thanks! Further tuning is still possible, but this helps a bit anyway at the moment.
